src/Importer.lua

```diff
@@ -105,29 +166,58 @@ function Importer:import(callingScript, path, exports)
 	assert(importCheck(callingScript, path, exports))
 
 	local parts = path:split("/")
-	local current =  callingScript
+	local current = callingScript
 	local hasAscendedParents = false
+	local isFirstPart = true
 
 	for _, pathPart in pairs(parts) do
-		local nextInstance = self:getNextInstance(current, pathPart, hasAscendedParents)
+		local nextInstance = self:getNextInstance(current, pathPart, hasAscendedParents, isFirstPart)
 
+		if isFirstPart then
+			assert(nextInstance, ("'%s' is not the name of a service, alias, or child of current dataModel (%s)")
+				:format(pathPart, self.dataModel.Name))
+		else
+			assert(nextInstance, ("Could not find a child '%s' at \"%s.%s\"")
+				:format(pathPart, current:GetFullName(), pathPart))
+		end
```

Comment:
Love this. I think this will help a lot with understanding what went wrong in a path